services/callback/src/test/java/app/coronawarn/server/services/callback/controller/CallbackControllerTest.java

```diff
@@ -21,6 +21,7 @@
 import org.springframework.boot.test.mock.mockito.SpyBean;
 import org.springframework.http.ResponseEntity;
 import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.ActiveProfiles;
```

Comment:
Unused

Reply:
Maybe it would be also good to have a ChgsCallbackControllerTest to see if Batches are saved with CHGS?